hs_core/hydroshare/users.py

```diff
@@ -332,13 +332,16 @@ def get_resource_list(creator=None, group=None, user=None, owner=None, from_date
                 coverages.add(coverage.id)
 
         for coverage in Coverage.objects.filter(type="point"):
-            coverage_shape = Point(
-                coverage.value.get('east', None),
-                coverage.value.get('north', None),
-            )
-
-            if search_polygon.intersects(coverage_shape):
-                coverages.add(coverage.id)
+            try:
+                coverage_shape = Point(
+                    coverage.value.get('east', None),
+                    coverage.value.get('north', None),
+                )
+
+                if search_polygon.intersects(coverage_shape):
+                    coverages.add(coverage.id)
+            except Exception as e:
+                pass
```

Comment:
I think it'd be good to add the exception messages into the log for later debug and diagnosis rather than just passing through?